govuk-one-login/authentication-frontend
# BAU: Make it easier (possible) to run integration tests locally

## What

- set REDIS_* in .env files: This means that when the file is loaded with dotenv, the values are correctly loaded
- Get envars with helpers in tests: we usually use the get* functions to get envars - it's easier to debug failures



## How to review



1. Check out this branch
2. Run integration tests
3. Observe that they run properly
